crates/bevy_window/src/window.rs

```diff
@@ -255,6 +255,27 @@ impl Default for Window {
 }
 
 impl Window {
+    /// Gets the window's current maximized state.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Web:** Unsupported.
+    pub fn is_maximized(&self) -> bool {
+        self.internal.maximized_state
+    }
+
+    /// Gets the window's current minimized state.
+    ///
+    /// If the minimized state can't be determined, this value will be `None`.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Wayland:** always None.
+    /// - **iOS / Android / Web:** Unsupported.
+    pub fn is_minimized(&self) -> Option<bool> {
```

Comment:
Similar question here, what happens on unsuported platforms?
Should we return None?